routes/users.router.js

```diff
@@ -0,0 +1,103 @@
+const express = require("express");
+const router = express.Router();
+const { extend } = require("lodash");
+const { User } = require("../models/user.model");
+
+router.route("/").post(async (req, res) => {
```

Comment:
Since you are doing signup here, you should move this to a signup route instead of keeping it on the base route of this router.

Reply:
yeah, makes sense, will do it.